### PR TITLE
added Head of Sales ad in DE and En

### PR DESCRIPTION
**Pull request by jutta-frieden**
Removed Account Manager ad and replaced with Head of Sales ad Germany.

### CODE REVIEW

**Comment by jutta-frieden** [comment]
@georg911 Hi Georg, can you have a look through the German copy and possibly also UK copy?
@Ken-Lo Could you help me with the code please?

File: app/pages/about/jobs/positions/head-of-sales-germany.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+            Lead our growth and help us build our presence in Germany
+          </p>
+          <p className='para'>
+          GoCardless is innovating on top of the SEPA network across Europe and equivalents around the world.
+          By making these networks easier to integrate into modern internet services and adding intelligent,
+          powerful and flexible capabilities we are adding value in completely new ways. GoCardless will continue
+          to disrupt these systems as payment mechanisms continue to evolve.
+          </p>
+          <p className='para'>
+            We're looking for a Head of Sales to help us grow our presence in the Germany market.
```
Comment (georg911) [inline app/pages/about/jobs/positions/head-of-sales-germany.js]: "in the German market." @jutta-frieden

**Comment by georg911** [comment]
all :+1: from my side with the 1 comment @jutta-frieden  :bouquet:

File: app/pages/about/jobs/positions/head-of-sales-germany.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        <Translation locales='de'>
+          <h3 className='u-text-heading-light u-text-m u-color-dark-gray'>Head of Sales Deutschland</h3>
+          <p className='para'>
+            Finden neuer Geschäftsbeziehungen und Ausbauen unseres deutschen Teams
+          </p>
+          <p className='para'>
+            GoCardless betreibt Innovation auf dem bestehenden europäischen SEPA Zahlungsnetzwerk sowie vergleichbarer
+            Netzwerke auf der ganzen Welt. Wir machen diese Netzwerke einfacher zugänglich für digitale online Services,
+            fügen intelligente, flexible und verlässliche Funktionen hinzu, und schaffen somit Mehrwert für
+            unsere Kunden in einer ganz neuen Dimension. Tausende von Unternehmen in Großbritannien vertrauen uns bereits
```
Comment (georg911) [inline app/pages/about/jobs/positions/head-of-sales-germany.js]: Vielleicht: "Tausende britische Unternehmen" ? @jutta-frieden

**Comment by jutta-frieden** [comment]
@gabrielamatias the PR is in process. Adding you here so you stay up to date for the Greenhouse link.

**Comment by gabrielamatias** [comment]
The link is on! :)

**Comment by jutta-frieden** [comment]
@jamesshedden as discussed, could you have a look at 'out-of-date' message please? Thanks!

**Pull request merged**